bin/eslint.js

```diff
@@ -43,6 +43,8 @@ function buildIssueJson(message, path) {
   if(message.fatal) {
     checkName = "fatal";
   }
+  var line = message.line || 1;
```

Comment:
This reduced the complexity of the function to be under the 6 specified in the project `.eslintrc`